DeviceAdministration/Web/WebApiControllers/TelemetryApiController.cs

```diff
@@ -205,7 +207,9 @@ public class TelemetryApiController : WebApiControllerBase
                     var resultsModel = new AlertHistoryResultsModel();
 
                     IEnumerable<AlertHistoryItemModel> data =
-                        await _alertsLogic.LoadLatestAlertHistoryAsync(currentTime.AddMinutes(-CautionAlertMaxMinutes));
+                        await _alertsLogic.LoadLatestAlertHistoryAsync(
+                            currentTime.Subtract(CautionAlertMaxDelta), 
+                            MaxHistoryItems);
```

Comment:
Param is minResults, but this is passing in MaxHistoryItems.  Should fix name so it makes sense.